packages/data-stores/src/site/types.ts

```diff
@@ -219,6 +219,7 @@ export interface SiteDetailsOptions {
 	was_created_with_blank_canvas_design?: boolean;
 	woocommerce_is_active?: boolean;
 	wordads?: boolean;
+	launchpad_view: boolean | 'off' | 'full' | 'minimized';
```

Comment:
If I'm reading this right, it would allow boolean (true/false) or one of the specified strings. Just wondering if we want to be consistent about the allowed values here? In the endpoint handling for the /settings endpoint (D85918-code), I think we're expecting a string and one of the string value, not a boolean.

Reply:
Looking at this a second time, a more specific way to represent this would be:

```
launchpad_view: false | 'off' | 'full' | 'minimized';
```

The `false` value is returned when no `launchpad_view` option is found, and represents the empty case where the user has never gone through launchpad onboarding.